examples/astro/src/pages/web-components.astro

```diff
@@ -267,6 +268,62 @@ import Layout from '../layouts/Layout.astro';
           });
         });
       </script>
+  
+      <h2 class="ds-text-heading--2xl">Modal Dialog example</h2>
+      <ds-button id="modal-toggle">Open Modal</ds-button>
+      <ds-modal-dialog backdrop-click-exits="true" alert="false" size="wide">
+        <div>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed accumsan diam vitae metus
+            lacinia, eget tempor purus placerat.
+          </p>
+          <div slot="heading">
+            <h2>This is the modal heading.</h2>
+          </div>
+          <div slot="actions">
+            <ds-button id="submit">Submit (and close the dialog)</ds-button>
```

Comment:
I tried to close this example using the keyboard and was not able to do so after tabbing to the "Open Modal" button, pressing "enter" and then tabbing this submit button into focus and then pressing "spacebar" or "enter. Do you know why that's the case @jack-ryan-nava-pbc?